### ISSUE TITLE
[9.0 + 9.0E] Database anonimization doesn't work - ProgrammingError

### ISSUE THREAD

**Issue by Yenthe666** [odoo/odoo#10457]
**ProgrammingError when trying to anonimise database**

Impacted versions:

 - 9.0 + 9.0E

Steps to reproduce:

 1. Install the module Database Anonymization
 2. Go to settings > Database Anonymization > Anonymize database
 3. Click on 'Anonymize database'
<img src="http://i.imgur.com/zYwoQIR.png" alt="Info screen"/>

Current behavior:
 - ProgrammingError
```
  File "/home/odoo/runbot/extra/runbot/static/build/125471-9-0-aad21c/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/125471-9-0-aad21c/openerp/addons/anonymization/anonymization.py", line 402, in anonymize_database
    cr.execute(sql)
  File "/home/odoo/runbot/extra/runbot/static/build/125471-9-0-aad21c/openerp/sql_db.py", line 139, in wrapper
    return f(self, *args, **kwargs)
  File "/home/odoo/runbot/extra/runbot/static/build/125471-9-0-aad21c/openerp/sql_db.py", line 215, in execute
    res = self._obj.execute(query, params)
ProgrammingError: column "name" does not exist
LINE 1: select id, name from res_users
```

Expected behavior:

 - Database is anonymized

**Comment by Yenthe666** [comment]
Hi @mart-e and @odony this issue is still present in the latest runbot from today:
Apparently nobody tried anonimyzing the database in a few months? :wink: 
![image](https://cloud.githubusercontent.com/assets/6352350/15470862/3e0b469c-20f3-11e6-9264-3bea54f352e3.png)

**Comment by mart-e** [comment]
Fixed at #11806

**Comment by mart-e** [comment]
ec005af [FIX] anonymization: default anonymization fields - Simon Leblanc
45483b9 [FIX] anonymization: allow anonymization on HTML fields - Simon Leblanc
42aa79d [CLA] Sign CLA of leblanc-simon - Simon Leblanc
